pipeline/mw-pipeline-v0.1.yaml

```diff
@@ -0,0 +1,253 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: mw-pipeline
+spec:
+  params:
+    - name: QUAY_REPO
+      type: string
+    - name: URL
+      description: Repository URL to clone from
+      type: string
+    - name: REVISION
+      description: 'Revision to checkout. (branch, tag, sha, ref, etc...)'
+      type: string
+      default: main
+    - name: BUILD_SCRIPT
+      description: 'The build script to embed with the Containerfile'
+      type: string
+      default: |
+        date
+    - name: RECIPE_IMAGE
+      type: string
+    - name: BUILD_TOOL
+      type: string
+    - name: BUILD_TOOL_VERSION
+      type: string
+    - name: JAVA_VERSION
+      type: string
+    - name: MVN_REPO_DEPLOY_URL
+      description: Maven repository to deploy to
+      type: string
+      default: http://indyhost/ #TODO remove default
+    - name: MVN_REPO_DEPENDENCIES_URL
+      description: Maven repository to get dependencies from
+      type: string
+    - name: ACCESS_TOKEN
+      type: string
+    - name: BUILD_ID
+      type: string
+    - name: caTrustConfigMapName
+      type: string
+    - name: DOMAIN_PROXY_TARGET_ALLOWLIST
+      type: string
+      default: "localhost,gateway.indy.corp.stage.redhat.com,cdn-ubi.redhat.com,repo1.maven.org,repo.scala-sbt.org,scala.jfrog.io,repo.typesafe.com,jfrog-prod-usw2-shared-oregon-main.s3.amazonaws.com"
+    - name: DOMAIN_PROXY_ENABLE_INTERNAL_PROXY
+      type: string
+      default: "true"
+    - name: DOMAIN_PROXY_INTERNAL_PROXY_HOST
+      type: string
+      default: "localhost"
+    - name: DOMAIN_PROXY_INTERNAL_PROXY_PORT
+      type: string
+      default: "8081"
+    - name: DOMAIN_PROXY_INTERNAL_PROXY_USER
+      type: string
+      default: "$(params.BUILD_ID)+tracking"
+    - name: DOMAIN_PROXY_INTERNAL_PROXY_PASSWORD
+      type: string
+      default: "$(params.ACCESS_TOKEN)"
+    - name: DOMAIN_PROXY_INTERNAL_NON_PROXY_HOSTS
+      type: string
+      default: "localhost,gateway.indy.corp.stage.redhat.com"
+    - name: ENABLE_INDY_PROXY
+      type: string
+    - name: INDY_PROXY_CLIENT_ID
+      type: string
+    - name: INDY_PROXY_CLIENT_CREDENTIAL
+      type: string
+    - name: PNC_KONFLUX_TOOLING_IMAGE
+      type: string
+      default: "quay.io/redhat-user-workloads/konflux-jbs-pnc-tenant/konflux-tooling:latest"
```

Comment:
I think we should get rid of the parameter `PNC_KONFLUX_TOOLING_IMAGE` and just use the released version in https://quay.io/repository/konflux-ci/pnc-konflux-tooling. This would require updates to the build driver.